daphne/src/vdaf/prio3.rs

```diff
@@ -52,9 +52,9 @@ pub(crate) fn prio3_shard(
             let vdaf = Prio3::new_count(2)?;
             Ok(shard!(vdaf, &measurement, nonce))
         }
-        (Prio3Config::Histogram { buckets }, DapMeasurement::U64(measurement)) => {
-            let vdaf = Prio3::new_histogram(2, buckets)?;
-            Ok(shard!(vdaf, &(measurement as u128), nonce))
+        (Prio3Config::Histogram { len }, DapMeasurement::U64(measurement)) => {
+            let vdaf = Prio3::new_histogram(2, *len)?;
+            Ok(shard!(vdaf, &(measurement as usize), nonce))
```

Comment:
The safe option would be `usize::try_from()`

Reply:
Ok, my new code will panic if `try_from` fails while converting a `u64` measurement on a 32-bit system.